### PR TITLE
Further changes to signed out help section

### PR DESCRIPTION
**Pull request by creativeappdevamit**
# What and why?
Further addition to signed out help
# How to test?
Frontstage UI test
# Trello
https://trello.com/c/YROMv1KW/503-s35-user-self-serve-signed-out-help-something-else
https://trello.com/c/Ia8UNUS0/504-s35-user-self-serve-signed-out-help-help-with-password

### CODE REVIEW

File: frontstage/templates/help/help-with-password-not-accept.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        "ariaLabel": "Breadcrumbs",
+        "id": "breadcrumbs",
+        "itemsList": breadcrumbsData
+    })
+}}
+{% endblock breadcrumb %}
+
+{% block main %}
+<h1 class="u-fs-xl">My new password isn't being accepted</h1>
+<p>You cannot include the special character "@" or "at" symbol, in your password.</p>
```
Comment (warrenbailey) [inline frontstage/templates/help/help-with-password-not-accept.html:32]: is this content right?

**Review by warrenbailey — APPROVED** [review]
some of the content doesn't look quite right but if it's been reviewed by design 👍

File: frontstage/templates/help/help-with-password-not-accept.html (hunk 1 above)
Comment (creativeappdevamit) [reply frontstage/templates/help/help-with-password-not-accept.html:32]: I believe so.. it went via the design review.

**Pull request merged**